client/src/styles/legacy/_style.scss

```diff
@@ -959,67 +959,6 @@ html, body {
   margin: 0 0 $spacer; // gets padding from tab panel
 }
 
-.notice-item {
```

Comment:
All those styles got copied in the Toasts SCSS file.

Reply:
We created custom toasts before bootstrap released their own which we called ‘notices’. Going forward and adding this component to the pattern library I think we should adopt bootstrap’s toasts instead of our own custom ones. This means we can align with the toasts bootstrap has to offer and features which we do not yet have implemented. https://getbootstrap.com/docs/4.5/components/toasts/ cc @silverstripeux

Reply:
Cool ... looks like it's in reactstrap as well. Not sure when it got added, so I might have to bump the version that we're shipping with admin 1.7.x. https://reactstrap.github.io/components/toasts/

Do you have a mock-up showing how it should look? Or is it meant to retrain the existing look and feel?

If you don't have a mock-up, I can probably just put it in with the vanilla look for now and we can go from there.

Reply:
FYI this is what it looks like right now. 
![image](https://user-images.githubusercontent.com/1168676/86546355-8c635900-bf88-11ea-96e9-22db1cd0f9ae.png)

Reply:
This is what we have currently in [DSM]( https://projects.invisionapp.com/dsm/silver-stripe/silver-stripe/asset/components/5dae8013fccf7c3293fb199d). I'll make sure to do a review in the coming week or so to make sure it's up to date.

Reply:
@maxime-rainville is that displaying just the title (without body text) here?

Reply:
That screen shot comes from my effort to exactly replicate the jQuery notice HTML. That screen shot doesn't use the bootstrap toast. That should be exactly how they look like right now.

I can get the boostrap toasts to look like that if you want to retain the existing look and feel.